backends-velox/src/main/scala/io/glutenproject/execution/ShuffledHashJoinExecTransformer.scala

```diff
@@ -87,7 +87,7 @@ case class ShuffledHashJoinExecTransformer(
    * Returns whether the build and stream table should be exchanged with consideration of build
    * type, planned build side and the preferred build side.
    */
-  override lazy val exchangeTable: Boolean = hashJoinType match {
+  override lazy val switchTable: Boolean = hashJoinType match {
```

Comment:
nit: `switchTable` -> `needSwitchTable`.

And switchTable confused me, maybe switchChild make more sense for me, join children does not always be table.

Reply:
Updated, thanks.